addons/website/views/website_templates.xml

```diff
@@ -36,7 +36,7 @@
 
 <template id="layout" name="Main layout" inherit_id="portal.frontend_layout">
     <xpath expr="//html" position="before">
-        <t t-set="editable_in_backend" t-value="edit_in_backend or ('website_published' in main_object.fields_get() and main_object._name != 'website.page')"/>
+        <t t-set="editable_in_backend" t-value="edit_in_backend"/>
```

Comment:
Why not. Did someone asked for this change? Although, would the variable "editable_in_backend" still useful in that case?

Reply:
Indeed the variable "editable_in_backend" is useless now, I will remove it.

Reply:
> Did someone asked for this change?

Ok, now I know why you did not answer, that's just buggy without it... :disappointed: I'll restore it alongside blog post button which is not worth nor consistent to remove. Confirmed with PDE.